frontend/src/static/js/v_summary.js

```diff
@@ -456,6 +444,7 @@ window.vSummary = {
       return null;
     },
     updateSortSelection() {
+      this.getOptionWithOrder();
       if (this.filterGroupSelection === 'groupByAuthors') {
         if (!this.sortWithinGroupSelection || this.sortingWithinOption === 'name') {
           this.sortWithinGroupSelection = 'searchPath asc';
```

Comment:
all these dun need change ah?

Reply:
Referring to line 450? 
`sortWithinGroupSelection` has to be updated directly. Updating `sortingWithinOption` or `isSortingWithinDsc` will not update the UI, as its not bound 2-way.

Reply:
makes sense, add comment `\\update UI`